llvm/test/Transforms/SLPVectorizer/AArch64/fminimumnum.ll

```diff
@@ -0,0 +1,516 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; RUN: opt --passes=slp-vectorizer --mtriple=aarch64 -mattr="+neon" -S < %s | FileCheck %s
+
+@input1_f32 = dso_local local_unnamed_addr global [9 x float] zeroinitializer, align 16
+@input2_f32 = dso_local local_unnamed_addr global [9 x float] zeroinitializer, align 16
+@output_f32 = dso_local local_unnamed_addr global [9 x float] zeroinitializer, align 16
+@input1_f64 = dso_local local_unnamed_addr global [9 x double] zeroinitializer, align 16
+@input2_f64 = dso_local local_unnamed_addr global [9 x double] zeroinitializer, align 16
+@output_f64 = dso_local local_unnamed_addr global [9 x double] zeroinitializer, align 16
+@input1_f16 = dso_local local_unnamed_addr global [9 x half] zeroinitializer, align 16
+@input2_f16 = dso_local local_unnamed_addr global [9 x half] zeroinitializer, align 16
+@output_f16 = dso_local local_unnamed_addr global [9 x half] zeroinitializer, align 16
```

Comment:
```suggestion
@input1_f32 = global [9 x float] zeroinitializer, align 16
@input2_f32 =global [9 x float] zeroinitializer, align 16
@output_f32 = global [9 x float] zeroinitializer, align 16
@input1_f64 = global [9 x double] zeroinitializer, align 16
@input2_f64 = global [9 x double] zeroinitializer, align 16
@output_f64 = global [9 x double] zeroinitializer, align 16
@input1_f16 = global [9 x half] zeroinitializer, align 16
@input2_f16 = global [9 x half] zeroinitializer, align 16
@output_f16 = global [9 x half] zeroinitializer, align 16
```

Can drop all the qualifiers on the globals (though in general I think the IR will be shorter if you just use function arguments for all these pointers)

Reply:
I have a try something like
`void fmin32(float input1_f32[9], float input2_f32[9], float output_f32[9])`
The result is even longer.